proto/ignition/msgs/serialized_map.proto

```diff
@@ -50,6 +50,10 @@ message SerializedStateMap
 
   /// \brief All the entities currently in the simulation.
   map<uint64, SerializedEntityMap> entities = 2;
+
+  /// \brief Indicates if there is any one time component change,
+  /// or if all changes are periodic.
+  bool is_periodic_change = 3;
```

Comment:
How about matching the ECM APIs and flipping this around to `has_one_time_component_changes`?

That should be `true` less often, and considering that protobuf doesn't send `false` values (the default), I think we'll get smaller messages more often.

Reply:
I agree that that is more natural.

I flipped from `has_one_time_component_changes` to `all_changes_are_periodic`, so a "edifice" primary sees all messages of a "dome" secondary as "one_time" (because the dome version is not filling that field, and the default value for booleans in proto3 is `false` and cannot be changed).

But probably that kind of backwards compatibility doesn't matter, and `has_one_time_component_changes` is clearer.

Reply:
> But probably that kind of backwards compatibility doesn't matter

Oh yeah, we don't support cross-talk across versions, so that should be ok.

Reply:
fixed in https://github.com/ignitionrobotics/ign-msgs/pull/119/commits/288861c343a6a15183901ada1c72866873e794d6